go/vt/vttablet/tabletmanager/vreplication/vplayer_flaky_test.go

```diff
@@ -1737,6 +1732,7 @@ func TestPlayerDDL(t *testing.T) {
 	pos1 := primaryPosition(t)
 	// The stop position must be the GTID of the first DDL
 	expectDBClientQueries(t, qh.Expect(
+		"/update _vt.vreplication set state='Stopped'",
```

Comment:
The first updates come from vreplicator which is unaware of batching (which later includes the same update).